internal/task/task/bs/create_volume.go

```diff
@@ -76,6 +77,13 @@ func NewCreateVolumeTask(curveadm *cli.CurveAdm, cc *configure.ClientConfig) (*t
 	subname := fmt.Sprintf("hostname=%s image=%s", hc.GetHostname(), cc.GetContainerImage())
 	t := task.NewTask("Create Volume", subname, hc.GetSSHConfig())
 
+	if len(cc.GetContainerImage()) == 0 {
```

Comment:
Actually, there is a default value for client container image, so maybe it's an useless branch.
BTW: 
* you should add a step to do checking, maybe `step.Lamba` is a good choice.
* what is the purpose of those changes.